### PR TITLE
[motion_monitor] adjust turn indicator

### PR DESCRIPTION
**Pull request by muritane**
issue:
* https://github.com/4am-robotics/orga/issues/4394

### CODE REVIEW

File: cob_light/common/include/turnIndicatorMode.h [hunk 1]
```diff
@@ -14,79 +14,70 @@
...
-      _pos = 0;
-      _dir = dir;
-      if(_num_leds%2 == 0)
-      {
-        _center_pos = _num_leds/2 - 1;
-        if(_dir < 0)
-        _center_pos--;
+    _colors.assign(_num_leds, color::rgba());
+    _dir = dir;
+    if (dir == -1) { // left
```
Comment (HannesBachter) [inline cob_light/common/include/turnIndicatorMode.h]: left turn is `dir == 1`

**Review by HannesBachter — CHANGES_REQUESTED** [review]

File: cob_light/common/include/turnIndicatorMode.h (hunk 1 above)
Comment (muritane) [reply cob_light/common/include/turnIndicatorMode.h]: dir == -1 for left is defined in the mode factory, the led indices should be reversed and led vis should be adjusted as it seems:
- https://github.com/ipa320/cob_driver/blob/kinetic_dev/cob_light/common/src/modeFactory.cpp#L161-L168
```
    case cob_light::LightModes::TURN_LEFT:
        mode.reset(new TurnIndicatorMode(color, colorO->getNumLeds(), -1, requestMode.priority, requestMode.frequency,
                                      requestMode.pulses, requestMode.timeout));
        break;

    case cob_light::LightModes::TURN_RIGHT:
        mode.reset(new TurnIndicatorMode(color, colorO->getNumLeds(), 1, requestMode.priority, requestMode.frequency,
                                      requestMode.pulses, requestMode.timeout));
```

**Review by HannesBachter — APPROVED** [review]
tested on `mrl-2` :+1:

**Review by benmaidel — APPROVED** [review]
LGTM

**Pull request merged**